src/translations/screens/subscreens/Login.ts

```diff
@@ -61,7 +61,7 @@ const LoginTexts = {
       'Log in with a one-time code sent to your phone',
     ),
     input: {
-      heading: _('Mobilnummer', 'phone number'),
+      heading: _('Mobilnummer', 'Phone number'),
```

Comment:
Nice! I've seen this so many times, but never when I was able to fix it, and then forgot 😅